<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>camera</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colorlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>color</string>
    </property>
   </widget>
   <widget class="QLabel" name="detectedlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="positionlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="gpioStatusLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>txt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
